tests/test_welltest_dpds.py

```diff
@@ -182,3 +178,23 @@ def test_get_weighted_avg_press_time_derivative_lag2():
     assert len(dpdspt_w_lag2) == 247
     assert dpdspt_w_lag2[0] == pytest.approx(0.43083638)
     assert dpdspt_w_lag2[-1] == pytest.approx(0.12729989)
+
+
+def test_gendata_vec(tmpdir):
+    mockfcont = """
+    Time\tdTime
+    (hr)\t(hr)
+    0\t0
+    1\t1
+    """
+    fileh = open("index.txt", "w")
```

Comment:
Use `Path("index.txt").write_text(somestring)`. That includes the context manager.